### DISCUSSION TITLE
Connecting BMS using isolated cables

### DISCUSSION THREAD

**Discussion by xury77** [Louisvdw/dbus-serialbattery#184] [General]
Everything work OK, but I had problem (I think) with the ground loop connections between DalyBMS and CerboGX because Daly Uart cable is not isolated, and finally my cable was broken with burned wires. Hopefully Cerbo still works. Now I'm afraid to connect it again. Someone has similar problem? I looking for any isolated uart converter, but not so many available.
Second question: Is possible to connect using VE.Direct interface? As I read that is fully isolated.

**Comment by Louisvdw** [comment]
There are a few isolated cables available. They are a bit more expensive but give you extra safety. Best would be to do a search on your local marketplace.
Did you perhaps connect the V+ power cable from the Daly? If you are not using an isolated cable then you should only connect the ground and the Rx/Tx wires. 
Ve.Direct will not work, except if your BMS is using RS232 (I don't think any Daly has a RS232 option). Most use TTL (on to which you can use a TTL->RS232) or RS485 connection

**Reply by xury77** [reply]
Yes. I did not connect V+ from Daly side. I used GND,Tx and Rx  pins. 
BTW Ve.direct is not rs232C but pure LVTTL Uart (3,3V level). But originally is 19200 baud rate. I know it because I straight connected Ve.Direct to Esp32 uart and its works well. But i think it will be hard to force use it with something else. 
If you have any smart shunt etc I can check it. To read from Ve.direct just connect to GND and crossed RX.TX, xet 19200 baud rate and nothing else. Data is pushed every one second in text mode.

**Comment by pau1phi11ips** [comment]
Surprised to hear this. I use the included UART cable with my Daly and Cerbo and no issues at all.
